docs/state/action-naming-conventions.md

```diff
@@ -0,0 +1,32 @@
+# `@daffodil` Redux Action Naming Conventions
+
+Any given action dispatched internally by Daffodil, or exposed by Daffodil in such a way that it exists in user-space, should be authored by the following Convention.
+
+```txt
+[packageName] An Action With Good Hygiene
+```
+
+For an overview on Action Hygiene, watch the [talk](https://www.youtube.com/watch?v=JmnsEvoy-gY&ab_channel=ng-conf) by @mikeryandev.
+
+
+## Examples 
+We've provided a few examples to illustrate the concepts above.
+
+### Correct
+
+```txt
+[@daffodil/category] Category Page Load Action
+```
+
+### Incorrect
+
+#### Bad Hygiene
+```txt
+[@daffodil/category] Load Action
+```
+
+#### Bad Convention
```

Comment:
I feel like the reason for this being wrong is still "Bad Hygiene", because you can't tell which package it comes from. "Bad Convention" seems vague.

I think an example of "Bad Convention" that isn't vague would be something like:
```
[Daff Category] Category Page Load Action
```
because it isn't really bad hygiene if "Daff Category" is how we're choosing to represent the package name. It just doesn't follow the convention we've set.

Reply:
87a81c11